homeassistant/components/thermostat/__init__.py

```diff
@@ -37,6 +38,7 @@
 ATTR_CURRENT_TEMPERATURE = "current_temperature"
 ATTR_AWAY_MODE = "away_mode"
 ATTR_FAN = "fan"
+ATTR_FAN_MIN_ON_TIME = "minutes"
```

Comment:
This should have the same value as the key for `fan_min_in_time` in the state attributes for the entity that have this attribute, to make it intuitive how to call the service.